### PR TITLE
Switch to speedy new Travis boxes

### PR DESCRIPTION
**Pull request by indirect**

### CODE REVIEW

**Comment by indirect** [comment]
@homu r+

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: ruby
+sudo: 9000
```
Comment (segiddins) [inline .travis.yml]: what exactly does this change?

Comment (indirect) [inline .travis.yml]: It switches our builds to the beta Travis builders running on Gogle Compute. They can finish a Bundler build in about 10-11 minutes, which is 2x the speed of the other builders.

Comment (segiddins) [inline .travis.yml]: Cool!

**Comment by segiddins** [comment]
@homu r+

**Comment by indirect** [comment]
Starting to think that Travis isn't running on the auto branch, and so homu will never succeed...

**Comment by segiddins** [comment]
Any reason this is into 2-0-dev instead of master?

**Comment by segiddins** [comment]
@homu r+

**Comment by indirect** [comment]
The beta infra doesn't have Ruby 1.8 on it. :/

**Comment by indirect** [comment]
Hmm seems like the tests must still not be running?

**Pull request merged**